src/lib/containers/LoginPage.tsx

```diff
@@ -9,12 +10,23 @@ export type LoginPageProps = {
 
 const LoginPage: React.FunctionComponent<LoginPageProps> = props => {
   const { ssoRedirectUrl, redirectUrl, sessionCallback } = props
+  const [isOnUsernameOrPasswordScreen, setIsOnUsernameOrPasswordScreen] =
+    React.useState<boolean | undefined>()
   const thisClass = 'login-panel-wrapper'
 
   return (
     <div className={'login-panel-wrapper-bg'}>
       <div id={thisClass} className={thisClass}>
         <div className={'login-panel panel-left'}>
+          {isOnUsernameOrPasswordScreen && (
+            <button
+              type="button"
+              className="back-button"
+              onClick={() => setIsOnUsernameOrPasswordScreen(false)}
+            >
+              <IconSvg icon="arrowBack" />
+            </button>
+          )}
```

Comment:
Can you do this to make it look closer to the designs?

- Move the button up one level (and adjust styling if needed. I think it would be OK to just give the button absolute positioning)
- Change the `arrowBack` and `arrowForward` icons in IconSvg to use the MUI Arrow<Back/Forward> instead of Arrow<Back/Forward>Ios component

Also, consider using the MUI IconButton component instead of a button. See ComplexMenu.tsx for an example

```suggestion
        {isOnUsernameOrPasswordScreen && (
          <button
            type="button"
            className="back-button"
            onClick={() => setIsOnUsernameOrPasswordScreen(false)}
          >
            <IconSvg icon="arrowBack" />
          </button>
        )}
        <div className={'login-panel panel-left'}>
```